### PR TITLE
Auth route signin - WIP 

### PR DESCRIPTION
**Pull request by AlinaGoaga**
Sending this as it's getting a bit bloated => not included: loading, checking the session (as we dont have webAuth, do we want to save the tokens in the localstorage and check if they are there/their validity as part of checking the session?), errors and their handling

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,15 +18,18 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
+    "@types/react-router-dom": "^5.1.5",
     "bootstrap": "^4.4.1",
+    "git": "^0.1.5",
```
Comment (WebMaestroFr) [inline package.json:23]: I don't think `git` is a dependency here.

File: package.json [hunk 2]
```diff
@@ -18,15 +18,18 @@
...
     "helix-js": "ssh://git@github.com:ori-edge/helix-js.git",
     "husky": "^4.2.3",
     "lint-staged": "^10.1.1",
-    "node-sass": "^4.14.1",
+    "node-sass": "^4.14.1z",
```
Comment (WebMaestroFr) [inline package.json:27]: `1z` ? Is this a typo ?

File: src/contexts/auth/Provider.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    helix.auth.removeToken("deploy");
+    helix.auth.removeToken("core");
+    helix.auth.removeToken("platform");
+  };
+
+  const submitLogin = (token: string) => {
+    // The below is meant to be:
+    // if (helix.checkToken("cluster",token)) =>used clusterAPI assuming that is the one which will check it;
+    // I am expecting a boolean to come back
+    if (true) {
```
Comment (WebMaestroFr) [inline src/contexts/auth/Provider.tsx:27]: Fair enough... Knowing that we have a `if (true)` in the code will keep me up at night, but at least you're sure I'll deliver a `checkToken` method ASAP !

Comment (AlinaGoaga) [reply src/contexts/auth/Provider.tsx:27]: Hahah you know why it's in there :p Looking forward to the method!

File: package.json (hunk 2 above)
Comment (AlinaGoaga) [reply package.json]: Yes. Removed

File: package.json (hunk 1 above)
Comment (AlinaGoaga) [reply package.json]: Removed also

**Pull request merged**